### PR TITLE
Dev promote

### PR DESCRIPTION
**Pull request by nrajkum2-uiuc**

### CODE REVIEW

File: docs/user_permissions.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        </Arrow>
+    </Container>
+    <Box borderColor="black" text="Auth Server"/>
+</Container>
+
+Users should be allowed to change the persmission levels of other users if the user has the correct clearance.
+Permission levels are associated with a role/title (ex. admin, general user).
+All users with the same title have the same permissions and vice-versa.
+
+The auth server maintains the userIDs and role/title associated with each user.
```
Comment (CodeBrew28) [inline docs/user_permissions.mdx]: how are these defined? what does the config file look like?

File: docs/arrow.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
```
Comment (CodeBrew28) [inline docs/arrow.js]: Try to be specific about what is being imported so we don't import more than what's necessary/
```
import React, { Component } from "react";
```

File: docs/box.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
```
Comment (CodeBrew28) [inline docs/box.js]: ^^same import comment as above

File: docs/container.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
```
Comment (CodeBrew28) [inline docs/container.js]: ^^ same as above

**Comment by HRimaw2** [comment]
Include the default config file and explain it in the Docs

**Pull request merged**